typescript/create-onchain-agent/README.md

```diff
@@ -13,61 +13,47 @@ Before using `create-onchain-agent`, ensure you have the following installed:
 
 ## Usage
 
-To use `create-onchain-agent`, simply run:
+### Quickstart Templates
+
+To create a new project from scratch:
 
 ```sh
 npm create onchain-agent@latest
 ```
 
-This command will guide you through setting up an onchain agent project by prompting for necessary configuration options.
-
-## Features
-
-- **Guided setup**: Interactive prompts help configure the project.
-- **Supports multiple AI frameworks**.
-- **Supports multiple blockchain networks**.
-- **Select your preferred wallet provider**.
-- **Supports a preconfigured Next.js project with React, Tailwind CSS, and ESLint**.
-- **Supports a preconfigured Model Context Protocol Server project**.
-- **Automates environment setup**.
-
-### **Setup Process**
-The CLI will prompt you for the following details:
-
-1. **Project Name**: The name of your new onchain agent project.
-2. **Package Name**: The npm package name (auto-formatted if needed).
-3. **Framework**: Choose from available AI frameworks.
-4. **Network**: Choose from available blockchain networks.
-5. **Chain ID**: Specify if using a custom network.
-6. **Wallet Provider**: Select a preferred method for wallet management.
-
-After answering the prompts, the CLI will:
+This command lets you choose between two project templates:
+- **Next.js Template**: A full-stack web application with React, Tailwind CSS, and ESLint
+- **MCP Template**: A Model Context Protocol server project
 
-- Generate the project structure.
-- Copy necessary template files.
-- Configure the selected settings.
-- Display next steps to get started.
+### Component Generation
 
-## Getting Started
-
-Once your project is created, navigate into the directory and install dependencies:
+To add AgentKit components to an existing project:
 
 ```sh
-cd my-project
-npm install
-```
+# Generate a custom wallet provider
+npm create onchain-agent wallet-provider
 
-Then, configure your environment variables:
+# Generate a custom action provider
+npm create onchain-agent action-provider
 
-```sh
-mv .env.local .env
+# Generate framework-agnostic AgentKit setup.
+npm create onchain-agent agentkit
+
+# Generate framework-specific agent creation
+npm create onchain-agent agent
```

Comment:
I think we'll want these commands to be part of a local CLI, instead of part of the `create` package. The `create` can bootstrap a project like it does today, and an additional CLI will be installed (call it, for example, `agentkit`). Then it would be used like so: `npx agentkit generate action-provider` or `npx agentkit generate wallet-provider`

Reason being `npm create` is generally understood to be a bootstrap command for entire projects, so I think will be conceptually confusing for this use case of generating specific chunks of code within an already bootstrapped project